src/index.js

```diff
@@ -33,6 +33,33 @@ if ( typeof global.window !== 'undefined')
 if ( typeof window !== 'undefined')
     window.WebDollar = exportObject;
 
+if ( process && !process.env.BROWSER && process.env.COLLECT_STATS ){
```

Comment:
@IonutAlexandruPanait is this condition correct? we don't want to it to activate on browser